src/pages/iou/IOUModal.js

```diff
@@ -429,7 +429,9 @@ class IOUModal extends Component {
                                                 onConfirm={this.createTransaction}
                                                 onSendMoney={this.sendMoney}
                                                 hasMultipleParticipants={this.props.hasMultipleParticipants}
-                                                participants={this.state.participants}
+                                                participants={this.props.hasMultipleParticipants
+                                                    ? this.state.participants
+                                                    : _.filter(this.state.participants, email => this.props.myPersonalDetails.login !== email.login)}
```

Comment:
Should this be done here, I think it could be moved along the lines here https://github.com/Expensify/App/blob/4c64bec3f38af925a01911155eab54feafc96cda/src/pages/iou/IOUModal.js#L110-L111

Reply:
Participants are updated during IOUParicipantsPage step. So, it cannot be moved here. We need to make use of updated participants list.

Reply:
`IOUParicipantsPage` will not be used for Room or group flow. So no issues there.

Reply:
Check and let me know!

Reply:
I don't think it's a good idea. It adds more complication.